src/webcomponents/commons/filters/go-accessions-filter.js

```diff
@@ -87,33 +83,33 @@ export default class GoAccessionsFilter extends LitElement {
         this.dispatchEvent(event);
     }
 
-    openModal(e) {
-        $("#go_ontologyModal").modal("show");
+    openModal(_e) {
```

Comment:
Instead of adding an underscore character to the event argument, I suggest removing the argument, as you are not using it :)